src/storekit.cs

```diff
@@ -995,21 +1001,33 @@ interface SKAdNetwork {
 		[Static]
 		[Async]
 		[Export ("startImpression:completionHandler:")]
-		void StartImpression (SKAdImpression impression, [NullAllowed] Action<NSError> completion);
+		void StartImpression (SKAdImpression impression, [NullAllowed] SKAdNetworkCompletionHandler completion);
 
 		[NoWatch, NoTV, NoMac]
 		[iOS (14,5)]
 		[MacCatalyst (14,5)]
 		[Static]
 		[Async]
 		[Export ("endImpression:completionHandler:")]
-		void EndImpression (SKAdImpression impression, [NullAllowed] Action<NSError> completion);
+		void EndImpression (SKAdImpression impression, [NullAllowed] SKAdNetworkCompletionHandler completion);
 
 		[NoWatch, NoTV, NoMac, iOS (15,4), MacCatalyst (15,4)]
 		[Static]
 		[Async]
 		[Export ("updatePostbackConversionValue:completionHandler:")]
-		void UpdatePostback (nint conversionValue, [NullAllowed] Action<NSError> completion);
+		void UpdatePostback (nint conversionValue, [NullAllowed] SKAdNetworkCompletionHandler completion);
+
+		[NoMac, iOS (16,1), MacCatalyst (16,1), NoWatch, NoTV]
+		[Static]
+		[Async]
+		[Export ("updatePostbackConversionValue:coarseValue:completionHandler:")]
+		void UpdatePostbackConversionValue (nint fineValue, [BindAs (typeof (SKAdNetworkCoarseConversionValue))] NSString coarseValue, [NullAllowed] SKAdNetworkCompletionHandler completion);
```

Comment:
It's better to name it like the existing method:

```suggestion
		void UpdatePostback (nint conversionValue, [BindAs (typeof (SKAdNetworkCoarseConversionValue))] NSString coarseValue, [NullAllowed] SKAdNetworkCompletionHandler completion);
```